pom.xml

```diff
@@ -440,6 +440,12 @@
             <version>${netty.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
I'm being nitpicky here, but you could move this so it's directly underneath the `junit` dependency. The rest of the dependencies of this file seem to be organised that way.